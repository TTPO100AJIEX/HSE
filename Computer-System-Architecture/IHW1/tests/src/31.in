100
-333 -18 -259 -293 -126 322 -29 137 81 -50 -269 257 -343 102 -18 46 179 -28 125 88 306 333 79 362 -259 -93 -181 -310 -113 -289 -80 191 -265 -239 -118 -6 225 -115 -171 -146 264 122 234 213 19 40 -156 358 290 102 91 224 181 -108 -13 -265 32 251 -360 97 6 -169 171 -13 27 304 71 68 8 73 -206 253 -324 92 -190 -337 329 -277 288 -318 35 -230 169 -44 160 -79 -61 -221 -41 292 238 -98 -256 165 175 -57 -124 308 330 -28