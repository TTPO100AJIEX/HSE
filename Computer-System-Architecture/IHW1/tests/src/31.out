-333 -18 -293 -126 322 -29 137 -269 257 -343 102 -18 46 179 -28 125 88 306 333 79 362 -259 -93 -310 -113 -289 -80 191 -265 -239 -118 -6 225 -171 -146 264 122 234 19 40 -156 358 91 224 -108 -13 -265 32 251 -360 97 -169 171 -13 27 304 8 73 -206 253 -324 92 -337 329 -277 288 -318 35 -230 169 -44 160 -79 -61 -221 -41 292 -256 165 175 -124 308 330 